README.md

```diff
@@ -21,6 +21,13 @@ Version:
 
 [![Krop Release](https://img.shields.io/github/release/tamimattafi/krop.svg?style=for-the-badge&color=darkgreen)](https://github.com/tamimattafi/krop/releases)
 
+Compatibility:
+
+| Krop version | Kotlin version | Compose version |
+|--------------|----------------|-----------------|
+| 0.1.5+       | 2.0.20+        | 1.7.0           |
+| 0.1.2        | 2.0.20         | 1.6.11          |
+
```

Comment:
I think we should add more versions
0.1.0 -> Kotlin 1.9.24 + Compose 1.6.11 
0.1.1...0.1.2 -> Kotlin 2.0.20 + Compose 1.6.11
0.1.3 -> Kotlin 2.0.21 + Compose 1.7.0-rc01

Also, we have the current versions since 0.1.4, so I think it should be like this:
0.1.4+ -> Kotlin 2.0.21 + Compose 1.7.0

Reply:
I agree about the `0.1.0`, that may be informative for people who are still on Kotlin 1.9. `0.1.3` does not seem relevant, since it's an RC version of compose. I'm still on `2.0.20`, which works well, even though the library uses `2.0.21`

Reply:
@MathiasDL I agree that including 0.1.3 will have minimal impact, but the versions are distinct from 0.1.4, and we can't group them in one cell, even if it's just an RC.

I think it would be relevant to some people, that don't want to have different artifact versions in their dependencies, or don't want to include libraries with RC or Beta versions of compose.

I'm not sure if I correctly presented my point of view 😅 for us, it's just one extra line, but for some users it's an important insight

Reply:
@MathiasDL Maybe we can use 1.9 and 2.0 for Kotlin, and 1.6 and 1.7 for compose (Without Z-patch), in this case we can group more versions together under X.Y

Reply:
Leaving out the patch makes sense, kotlin/compose will likely never do any breaking changes on patch versions

Reply:
Hello @MathiasDL!
Are you planning to update this PR?

Reply:
@tamimattafi I updated the table, let me know what you think

Reply:
@MathiasDL Looks good to me, thank you!